plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java

```diff
@@ -465,7 +465,7 @@ public FSDataInputStream open(Path path, int bufferSize)
     public FSDataOutputStream create(Path path, FsPermission permission, boolean overwrite, int bufferSize, short replication, long blockSize, Progressable progress)
             throws IOException
     {
-        // Ignore the overwrite flag, since Trino always writes to unique file names.
+        // Ignore the overwrite flag, since Trino Hive connector *usually* writes to unique file names.
```

Comment:
SHould it be rather "is expected to"?

Reply:
I think the intent is to document the current state, not the contract. There is no contract like that.

Reply:
Yeah - you are right. There are cases where we are not randomizing filenames.